development.env

```diff
@@ -0,0 +1,2 @@
+BACKEND_URL    = http://localhost:3000/api/ 
+STRIPE_API_KEY = pk_test_51OhTCmDf0QjduFWJEpbxu9RBle3CZTnl7uDWLD68F2nu44nsLPEAtIawuGiqf4T6IJrK1o10Dnhy1FwsmP8dooms00HxIfUBe1
```

Comment:
**Security Issue**: Hard-coded API keys in environment files can be inadvertently exposed through version control systems. <br> **Fix**: Remove the hard-coded Stripe API key and replace it with a secure method of injecting the key at runtime. <br> **Code Suggestion**: 
 ```
 -STRIPE_API_KEY = pk_test_51OhTCmDf0QjduFWJEpbxu9RBle3CZTnl7uDWLD68F2nu44nsLPEAtIawuGiqf4T6IJrK1o10Dnhy1FwsmP8dooms00HxIfUBe1
 +STRIPE_API_KEY = ${{ secrets.STRIPE_API_KEY }}
 ```